themes/delphi/assets/css/layouts.scss

```diff
@@ -0,0 +1,38 @@
+$grid-gap: (32/1440)*100%;
+
+.content-grid {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-gap: $grid-gap;
+}
+
+.grid-2-8 {
+    grid-column: 2 / 8;
+  }
+  .grid-8-12 {
+    grid-column: 8 / 12;
+  }
+  .grid-3-11 {
+    grid-column: 3 / 11;
+  }
+  .grid-2-12 {
+    grid-column: 2 / 12;
+  }
+  .grid-1-2 {
+    grid-column: 1 / 2;
+  }
+
+@media only screen and (max-width: $breakpoint-small) {
+    .content-grid {
+        display: block; 
+    }
+}
+
+.content-row {
+    display: flex;
+    flex-direction: row;
+}
+
+.content-row .content{

```

Comment:
Ah, this would conflict with the top-level `content` style.  I'll rename this to `row-content`.  But we should also be able to use the normal uikit grid instead of .content-row.

Reply:
so `.content-row .row-content`?

Reply:
Yeah, `.content-cell` would be a better name.